code/game/area/almayer.dm

```diff
@@ -393,6 +390,19 @@
 	icon_state = "starboard"
 	fake_zlevel = 2 // lowerdeck
 
+//new hallways areas
+
+/area/almayer/hallways/upper
+	fake_zlevel = 1 // upperdeck
+
+/area/almayer/hallways/upper/port
+	name = "\improper Upper Deck port Hallway"
+	icon_state = "port"
+
+/area/almayer/hallways/upper/starboard
+	name = "\improper Upper Deck starboard Hallway"
```

Comment:
Capitalize Starboard and Port to keep it similar in style to how it is currently.

Reply:
done